microprofile/src/test/java/io/smallrye/dux/microprofile/TestMpConfigProvider.java

```diff
@@ -0,0 +1,38 @@
+package io.smallrye.dux.microprofile;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.spi.ConfigBuilder;
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+
+public class TestMpConfigProvider extends ConfigProviderResolver {
```

Comment:
Can we rename this to `TestMicroProfileProvider`?

Reply:
I renamed to TestMicroProfileConfigProvider